.travis.yml

```diff
@@ -172,8 +172,11 @@ jobs:
       name: "Pushing release to hub.docker.com"
     - script:
       - bin/s3 sync s3://travis-$TRAVIS_BUILD_NUMBER/build-artifacts build/package
-      - bin/release_android "$TRAVIS_TAG" "$BINTRAY_USER" "$BINTRAY_TOKEN"
-      name: "Publish Android SDK to Bintray and JCenter"
+      - openssl aes-256-cbc -K $encrypted_2698a83519b3_key -iv $encrypted_2698a83519b3_iv -in bin/package/android/key.gpg.enc -out bin/package/android/key.gpg -d
+      - gpg --import bin/package/android/key.gpg
+      - gpg --list-keys
+      - bin/release_android "$TRAVIS_TAG"
```

Comment:
- Script could ask to inject variable for GPG signing key 
- Script names does not match with `release_android_sonatype`, please align

Reply:
The script is called `bin/release_android`.